flytekit/plugins/__init__.py

```diff
@@ -34,7 +34,7 @@
 _lazy_loader.LazyLoadPlugin(
     "schema",
     [
-        "numpy>=1.14.0,<2.0.0",
+        "numpy>=1.14.0,<1.19.0",
```

Comment:
Please revert this. This PR has been approved: https://github.com/lyft/flytekit/pull/144